src/fsharp/FSharp.Core/list.fs

```diff
@@ -523,8 +528,7 @@ namespace Microsoft.FSharp.Collections
 
         [<CompiledName("SortDescending")>]
         let inline sortDescending list =
-            let inline compareDescending a b = compare b a
-            sortWith compareDescending list
+            sortWith compare list
```

Comment:
This looks like it will do a regular sort rather than a descending sort

Reply:
Oups. Will revert that

Reply:
But first let's see if a test catches it ;-)

Reply:
But first let's see if a test catches it ;-)

Reply:
ok fixed and indeed a test was catching it.